zoo/analytics/models.py

```diff
@@ -65,7 +65,7 @@ class Meta:
     major_version = models.BigIntegerField(null=True, blank=True)
     minor_version = models.BigIntegerField(null=True, blank=True)
     patch_version = models.BigIntegerField(null=True, blank=True)
-    version = models.CharField(max_length=100, null=True, blank=True)
+    version = models.CharField(max_length=200, null=True, blank=True)
```

Comment:
I guess when we chose 100 characters it was pretty arbitrary. Do you have some better reason for choosing 200? 😅 
Or will we use this until the next time we need a bigger string?

Reply:
I do think 200 should be enough but I'm not 100% sure. One more thing we can do is change add some truncate loginc to zoo/analytics/tasks/repo_analyzers.py run_all function ?

Reply:
Will leave like this for now.